src/contactInfo/infra/repositories/ContactRepository.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiRepository } from '../../../core/infra/repositories/ApiRepository'
+import { Contact } from '../../domain/models/Contact'
+import { IContactRepository } from '../../domain/repositories/IContactRepository'
+import { ContactDTO } from '../../domain/dtos/ContactDTO'
+
+export class ContactRepository extends ApiRepository implements IContactRepository {
+  public async submitContactInfo(contactDTO: ContactDTO): Promise<Contact[]> {
+    return this.doPost(`/admin/feedback`, contactDTO)
```

Comment:
I don't think we should use this API endpoint. It's an endpoint under the 'admin' basepath. These calls are restricted to administrators only, and AFAIK, they are limited to calls within the same host (they would fail if the consumer and the backend are not on the same host). Additionally, the call returns data that could be sensitive, such as the recipient's email address. Also, please take a look at the related comment in the endpoint definition https://github.com/IQSS/dataverse/blob/develop/src/main/java/edu/harvard/iq/dataverse/api/FeedbackApi.java#L33.

Reply:
Thanks for reviewing and pointing out this, Guillermo. Sorry I ignored that this is under /admin. 
Since this is the only API endpoint related to the contacts from [native API](https://guides.dataverse.org/en/6.4/api/native-api.html#send-feedback-to-contact-s), it would make sense to create a new issue in the Dataverse repository?